e2e-tests/adapters/gatsby-node.ts

```diff
@@ -62,3 +63,128 @@ export const createPages: GatsbyNode["createPages"] = ({
     context: {},
   })
 }
+
+// Image CDN
+export const createSchemaCustomization: GatsbyNode["createSchemaCustomization"] =
+  function createSchemaCustomization({ actions, schema, store }) {
+    actions.createTypes(`
+    type UnsplashImage implements Node & RemoteFile {
+      id: ID!
+    }
+
+    type DeployedLocalImage implements Node & RemoteFile {
+      id: ID!
+    }
+  `)
+
```

Comment:
this should be removed in favor of MyRemoteFile (which is copied from e2e-tests/production)
```suggestion
```